app/graphql/types/query_type.rb

```diff
@@ -1,4 +1,12 @@
 class Types::QueryType < Types::BaseObject
+  field :my_account, Types::Account, null: true do
```

Comment:
`current_account`? `my_account` feels kinda odd when you're doing auth for other users.

Reply:
`me` lololol

Reply:
I agree with wopian, `current_account` is probably a better phrasing

Reply:
ok